### ISSUE TITLE
Review Issues

### ISSUE THREAD

**Issue by sadrasabouri** [Sharif-SLPL/unsupervised-style-transformation#2]
I'm testing your code in [this](https://colab.research.google.com/drive/1tZOU6Fgxz8Gu3i_ake6qJccltwgMZPZn) google colab file which is going to be part of this code meanwhile. I just shared it with your primary email address. Feel free to change it.
**Finally we should have a `ipynb` like this which illustrates your code routine to make it more reproducible.**
<hr>
Bellow are some minor issues needed to be solved for proceeding:

- [x] [Here](https://github.com/Sharif-SLPL/unsupervised-style-transformation/blame/master/README.md#L27) you mentioned we should change `BASE_CONFIG/local_model_path` to point to our model.

- which model should this variable point to?
- how should we download it?

- [x] [Here](https://github.com/Sharif-SLPL/unsupervised-style-transformation/blob/master/requirements.txt#L68) you used an uncommon requirement. Please check this requirement and change it to proper value. Now when I'm `pip install -r requirements.txt`ing I got bellow error:
```
Processing /dlib-19.18.0-cp37-cp37m-linux_x86_64.whl
ERROR: Could not install packages due to an OSError: [Errno 2] No such file or directory: '/dlib-19.18.0-cp37-cp37m-linux_x86_64.whl'
```

**Comment by erfan226** [comment]
Great! I will check it and modify parts that need to be changed. 
About the `BASE_CONFIG/local_model_path`, this points to the trained model by the user (or the one shared by us outside of HuggingFace). If no model is found in the specified path, the loaded model would be from `BASE_CONFIG/model_path`. So it's not necessary to download anything.
Oh, about the requirements, I haven't tested it on another environment. It's strange that it has caused an error on Colab! I will look into it.

**Comment by erfan226** [comment]
Fixed in c92600bab54e1db7547f507dd36cd335ee790669
